lib/request.js

```diff
@@ -21,7 +21,18 @@ module.exports = (options) => {
     const httpModule = options.protocol === 'https:' ? https : http;
     return new Promise((resolve, reject) => {
         let hasRequestEnded = false;
+        const startTime = Date.now();
```

Comment:
Would be nice to not initialize this `startTime` and `timings` object if corresponding option is not set